### PR TITLE
DOC-576-change-logical-partitions-to-pre-replication

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2730
Review deadline: Wed Sept 18

## Page previews

- [BYOC tiers](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/byoc-tiers/) and [Dedicated tiers](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/dedicated-tiers/)
- Control Plane API [BYOC](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/manage/api/cloud-byoc-controlplane-api/#cluster-tiers) and [Dedicated](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/manage/api/cloud-dedicated-controlplane-api/#cluster-tiers)
- [Serverless](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/serverless/)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by bpraseed** [comment]
@micheleRP - is it difficult to add pre-replication to the column header so its easy to read and not have to refer to notes section ?

**Comment by micheleRP** [comment]
> @micheleRP - is it difficult to add pre-replication to the column header so its easy to read and not have to refer to notes section ?

See [BYOC with Partitions, pre-replication](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/byoc-tiers/) or [Dedicated with Partitions (pre-replication)](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/dedicated-tiers/). Resizing the screen width works better with the BYOC version. Do you care which one?

File: modules/get-started/pages/cluster-types/serverless.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ Each Serverless cluster has the following limits:
 
 * Ingress: up to 10 MBps, 0.5 MBps guaranteed
 * Egress: up to 30 MBps, 1.5 MBps guaranteed
-* Partitions: 100 logical partitions
+* Partitions: 100 partitions
```
Comment (vladoschreiner) [inline modules/get-started/pages/cluster-types/serverless.adoc:25]: Could we please mention (as a comment or note) that this limit doesn't include partition replicas?

Comment (micheleRP) [reply modules/get-started/pages/cluster-types/serverless.adoc:25]: @vladoschreiner I've updated the [note following Serverless limits.](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/serverless/#limits)

**Comment by bpraseed** [comment]
> See [BYOC with Partitions, pre-replication](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/byoc-tiers/) or [Dedicated with Partitions (pre-replication)](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/dedicated-tiers/). Resizing the screen width works better with the BYOC version. Do you care which one?

I liked the Dedicated version better

**Comment by micheleRP** [comment]
> > See [BYOC with Partitions, pre-replication](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/byoc-tiers/) or [Dedicated with Partitions (pre-replication)](https://deploy-preview-70--rp-cloud.netlify.app/redpanda-cloud/reference/tiers/dedicated-tiers/). Resizing the screen width works better with the BYOC version. Do you care which one?
> 
> I liked the Dedicated version better

Done

File: modules/reference/partials/tiers.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ ifdef::env-byoc[]
 When you create a BYOC cluster, you select your usage tier. Each tier provides tested workload configurations for maximum throughput, partitions, and connections. 
 
 |=== 
-| Tier | Ingress | Egress | Logical partitions | Connections
+| Tier | Ingress | Egress | Partitions (pre-replication) | Connections
```
Comment (Feediver1) [inline modules/reference/partials/tiers.adoc:7]: Is there a way to put a line break in the Partitions (pre-replication) heading cell? The break in the output is a bit unfortunate/distracting. Or, perhaps footnote it and put the pre-replication caveat there?

File: modules/reference/partials/tiers.adoc [hunk 3]
```diff
@@ -21,7 +21,7 @@ When you create a BYOC cluster, you select your usage tier. Each tier provides t
 ====
 * On Azure, tiers 1-3 are supported. 
 * Depending on the workload, it may not be possible to achieve all maximum values. For example, a high number of partitions may make it more difficult to reach the maximum value in throughput.
-* Logical partition counts assume the Redpanda Cloud default replication factor of 3. If you set a higher replication factor, the maximum value for partitions will be lower.
+* Partition counts assume the Redpanda Cloud default replication factor of 3. If you set a higher replication factor, the maximum value for partitions will be lower.
```
Comment (Feediver1) [inline modules/reference/partials/tiers.adoc:24]:
```suggestion
* Partition counts are based on the assumption that the Redpanda Cloud default replication factor is 3. If you set a higher replication factor, the maximum value for partitions will be lower.
```

File: modules/reference/partials/tiers.adoc [hunk 4]
```diff
@@ -111,7 +111,7 @@ When you create a Dedicated cluster, you select your usage tier. Each tier provi
...
```
Comment (Feediver1) [inline modules/reference/partials/tiers.adoc:114]:
```suggestion
* Partition counts are based on the assumption that the Redpanda Cloud default replication factor is 3. If you set a higher replication factor, the maximum value for partitions will be lower.
```

**Review by Feediver1 — APPROVED** [review]
See feedback/suggestions

**Comment by alexlavoie-redpanda** [comment]
Have we considered having a column that calls out total replicas too? The reason being, my customer's confusion lies here:

**_Based on the section of the docs, would it be valuable to have a column for total replicas as well? In the case of tier 1 BYOC, 3000, because of the calculations assumed 3 replicas? I believe thats how I read the previous docs. ie: If I had a replication factor of 5 on all my topics my total partitions limit would actually be 3000 / 5 => 600_**

**Comment by micheleRP** [comment]
> Have we considered having a column that calls out total replicas too? The reason being, my customer's confusion lies here:
> 
> **_Based on the section of the docs, would it be valuable to have a column for total replicas as well? In the case of tier 1 BYOC, 3000, because of the calculations assumed 3 replicas? I believe thats how I read the previous docs. ie: If I had a replication factor of 5 on all my topics my total partitions limit would actually be 3000 / 5 => 600_**

@bpraseed please confirm what you want here!

**Comment by micheleRP** [comment]
> > Have we considered having a column that calls out total replicas too? The reason being, my customer's confusion lies here:
> > **_Based on the section of the docs, would it be valuable to have a column for total replicas as well? In the case of tier 1 BYOC, 3000, because of the calculations assumed 3 replicas? I believe thats how I read the previous docs. ie: If I had a replication factor of 5 on all my topics my total partitions limit would actually be 3000 / 5 => 600_**
> 
> @bpraseed please confirm what you want here!

We'll track adding total replicas in this issue: https://redpandadata.atlassian.net/browse/DOC-595

**Pull request merged**